cni-plugin/deployment/scripts/install-cni.sh

```diff
@@ -298,6 +304,11 @@ monitor() {
           cni_conf_sha="$(sha256sum "$directory/$filename" | while read -r s _; do echo "$s"; done)"
         fi
       fi
+    done &
```

Comment:
We probably need to be careful with backgrounding these tasks -- we want to ensure that `inotifywait` failures are propagated so that the program can fail if any of these calls fail.

There are other changes we probably need to make to this script first, but we'll want to ensure that we preserve PIDs and `wait` on them to surface errors.